legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -27,34 +27,43 @@
 from legal_api.utils.datetime import datetime as dt
 
 from legal_api.core.filing import Filing as coreFiling  # noqa: I001
-from .common_validations import validate_share_structure, validate_party_name  # noqa: I001
+from .common_validations import validate_court_order, validate_share_structure, validate_party_name  # noqa: I001
 from ... import namex
 from ...utils import get_str
 
 
-def validate(incorporation_json: Dict):  # pylint: disable=too-many-branches;
+def validate(incorporation_json: dict):  # pylint: disable=too-many-branches;
     """Validate the Incorporation filing."""
     if not incorporation_json:
         return Error(HTTPStatus.BAD_REQUEST, [{'error': babel('A valid filing is required.')}])
-    legal_type = get_str(incorporation_json, '/filing/business/legalType')
     msg = []
 
+    legal_type_path = '/filing/incorporationApplication/nameRequest/legalType'
+    legal_type = get_str(incorporation_json, legal_type_path)
+    if not legal_type:
+        msg.append({'error': babel('Legal type is required.'), 'path': legal_type_path})
+        return msg  # Cannot continue validation without legal_type
+
```

Comment:
Making this change since the api users may not provide business object in filing as it is not required as per the schema.